src/com/company/onCrease.java

```diff
@@ -0,0 +1,15 @@
+package com.company;
+
+public class onCrease {
+    int playerID;
+    int runs;
+    int ballFaced;
+    int inAt;
+    public onCrease(int id, int rn, int bf, int at)
```

Comment:
same comment here of parameter name as in setter/getter

Reply:
changed